doc/Uplink.xml

```diff
@@ -202,19 +202,22 @@
     <section>
       <title>Authentication</title>
       <para>Note that for the following discussion the roles of client and server are swapped.</para>
-      <para>The remote client shall authenticate itself using a valid server certificate. The service shall verify the validity of the remote certificate according to RFC 6125. </para>
-      <para>The service shall authenticate itself at the remote client using TLS client authentication according to RFC 5246 or subsequent specifications.</para>
-      <para>To uniquely identify local service on remote client, it is recommended to have a unique client certificate installed on each local service. For example, CN field or Serial number of installed certificate could be used to uniquely identify the local service.</para>
-      <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is
-        configured for the configuration without the need of further authentication request headers. </para>
-    </section>
-    <section>
-      <title>HTTP/2 Frames</title>
-      <para>Once an http/2 connection has been established the communication parameters are negotiated as specified by RFC 7540 so that the uplink can be used to exchange frames between the remote client and the local service.</para>
-    </section>
-    <section>
-      <title>HTTP Transactions</title>
-      <para>The uplink shall be used in reverse direction for http requests and responses. The remote client shall send requests that are served in a standard http manner by the local service.</para>
+      <para>The remote client shall authenticate itself using a valid server certificate. The service shall verify the validity of the remote certificate according to RFC 6125.</para>
+      <section>
+        <title>mTLS authentication</title>
+        <para>When using mTLS authentication, the service shall authenticate itself at the remote client using TLS client authentication according to RFC 5246 or subsequent specifications.</para>
+        <para>To uniquely identify local service on remote client, it is recommended to have a unique client certificate installed on each local service. For example, CN field or Serial number of installed certificate could be used to uniquely identify the local service.</para>
+        <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers. </para>
+        <para>To use this authentication mechanism, the <literal>CertificateID</literal> shall be set in the <literal>UplinkConfiguration</literal>.</para>
+      </section>
+      <section>
+        <title>JWT authentication</title>
+        <para>When using JWT authentication, the local service shall provide a JWT token when sending the upgrade request.</para>
```

Comment:
Since we are passing the JWT as url params, this statement may needs to be rephrased, pls check.

Reply:
If it is normal HTTP request we can pass like that, as of now there is no standard way to send the Authorization header in the Upgrade request, more over most WebSocket libraires don't support this, sending it as url param is the standard approach as of now.

Reply:
For a browser application I agree that it is hard to add the authorization header. However as the device implementation connects to the uplink server it seems to be straight forward to provide the token as authorization bearer header.

Reply:
Is there any RFC to support this? I could not find any. More over not only browser, event standard WebSocket libraries like websocketcpp does't allow to pass authorization token like this, we may have to modify the opensource library to do it that way. One work around is passing the access token in the sub-protocol field.
 Is there any specific reason you want to pass it in the header, instead of URL Param ?

Reply:
@HansBusch , I am ok with both the approaches defined in RFC6750 (section 2.1 and 2.3).